ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -72,6 +93,36 @@ def __init__(self, tracer=None, config=None, service=None):
         except Exception:
             log.warning("Failed to load CODEOWNERS", exc_info=True)
 
+    def _check_enabled_features(self):
+        # type: () -> Tuple[bool, bool]
+        if not self._app_key:
+            return False, False
+        url = "https://api.datadoghq.com/api/v2/libraries/tests/services/setting"
+        _headers = {"dd-api-key": os.getenv("DD_API_KEY"), "dd-application-key": self._app_key}
```

Comment:
curious: why is one key read from the env and one from `self._app_key` ?

Reply:
Agreed, not sure there's a point to store `self._app_key` when we can just get it directly from `os.getenv(...)` if we're just going to read it once/twice at `CIVisibility.__init__()`.